src/server/rpc/procedures/thing-speak/thing-speak.js

```diff
@@ -137,7 +137,12 @@ thingspeakIoT.privateChannelFeed = function(id, numResult, apiKey) {
     }
 };
 
-//put together the data from feeds and channel metadata
+/**
+ * put together the data from feeds and channel metadata
```

Comment:
I would reword this description to be more informative, concise and fit more with the tone/style of the other descriptions.